main/parse.c

```diff
@@ -1772,6 +1775,26 @@ extern void makeFileTag (const char *const fileName)
 	}
 }
 
+static void makeEOFTag (const char *const fileName)
```

Comment:
Please check my commit at https://github.com/vhda/ctags/tree/masatake-add-total-lines-to-file-tag